src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String toString() {
+        return "Ball number: " + getNumber() + "\nBall color: " + getColor();
+    }
+
+    public void setBall(int number, String color) {
```

Comment:
Setters are usually done on one field.
What you do in this method must be done in the constructor.